src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no,   initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.  0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card-container" data-qa="card">
+      <div class="picture-container">
+        <img
+          class="picture-container__picture"
+          src="./images/imac.jpeg"
+          alt="Computer imac"
+        >
+      </div>
+      <div class="brief-description">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="product-code">
+        Код товара: 195434
+      </div>
+      <div class="stars-rewievs-block">
+        <div class="stars">
```

Comment:
add stars--4 modifier and styles for all the possible modifiers, so that number of active stars would change if you change modifier in html to stars--5